examples/pylink-rtt

```diff
@@ -152,4 +154,8 @@ def main(target_device, block_address=None):
 
 
 if __name__ == "__main__":
+    if len(sys.argv) < 2 or sys.argv[1] in ("-h", "--help"):
```

Comment:
Maybe we should use `argparse` here?  I think when originally written, it was just meant as an example, but in exposing it on the command-line, we might want to use something like `argparse` instead since it will have this parsing inherently.